frontend/assets/style/_typography.scss

```diff
@@ -37,8 +37,7 @@ body {
   font-style: normal;
   font-weight: normal;
   font-size: $size-4; // 14px
-  line-height: 1.3125rem; // 21px
-  letter-spacing: 0.1px;
+  line-height: 1.5; // 21/14
```

Comment:
Same here ;)

Reply:
`line-height` 1.3125rem is exactly the same as 1.5, so it was just a matter of readability.
`letter-spacing` was removed because it isn't needed - I confirmed with @mmbotelho.